go-controller/pkg/metrics/ovn_northd.go

```diff
@@ -0,0 +1,164 @@
+package metrics
+
+import (
+	"strings"
+	"time"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+)
+
+var (
+	ovnNorthdVersion       string
+	ovnNorthdOvsLibVersion string
+)
+
+func getOvnNorthdVersionInfo() {
+	stdout, _, err := util.RunOVNNorthAppCtl("version")
+	if err != nil {
+		return
+	}
+
+	// the output looks like:
+	// ovn-northd 20.06.0.86f64fc1
+	// Open vSwitch Library 2.13.0.f945b5c5
+	for _, line := range strings.Split(stdout, "\n") {
+		if strings.HasPrefix("ovn-northd ", line) {
+			ovnNorthdVersion = strings.Fields(line)[1]
+		} else if strings.HasPrefix("Open  vSwitch Library ", line) {
```

Comment:
2 blank spaces?? /Open  vSwitch/Open vSwitch/

Reply:
Good catch. Will fix.

Reply:
(these two values weren't getting populated for other reasons, that I was planning on to debug .. we are getting these values way too early and ovn-appctl calls were failing.)